src/main/java/org/opensrp/web/config/security/BasicAuthSecurityConfig.java

```diff
@@ -63,7 +63,7 @@ protected void configure(HttpSecurity http) throws Exception {
 		.mvcMatchers("/**").hasRole(Role.OPENMRS)
 		.and()
     		.csrf()
-    		.ignoringAntMatchers("/rest/**","/location/**");
+    		.ignoringAntMatchers("/rest/**","/location/**", "/multimedia/upload");
```

Comment:
This is  so as multimedia endpoints use custom headers to authenticate. Has the client side been modified to support normal authentication

Reply:
Not sure I understand. Could you rephrase this?

I did not have to make any changes on the client.